dart/tool/service.md

```diff
@@ -2393,7 +2394,7 @@ class Script extends Object {
   string source [optional];
 
   // A table encoding a mapping from token position to line and column.
-  int[][] tokenPosTable [optional];
+  int[][] tokenPosTable;
```

Comment:
@bkonyi - I've seen this return an `null` recently (likely when `source` is null?) I'll send a CL that updates the spec to make this optional.

Reply:
(to update: this is done)